src/manifest.json

```diff
@@ -8,7 +8,8 @@
   },
   "permissions": [
     "storage",
-    "alarms"
+    "alarms",
+    "scripting"
```

Comment:
The chrome store is going to be obnoxious about this. They will do a super long review. To make sure this is used. Probably like 2-3 weeks.

I wonder if the storage stuff is moved to the extension storage if we can get by without this. Not a huge deal either way.

Reply:
I'm going to just remove it for now.